.gitignore

```diff
@@ -117,7 +117,6 @@ dmypy.json
 # editor
```

Comment:
Just a nit, but the `[dev]` install should update to include `mkdocs` deps.

Reply:
This is a cool command though. TIL. 

```
python -k mkdocs get-deps
```

Reply:
Yep I forgot to substitute that in both _setup.py_ and _docs-requirements.txt_